_i18n/es/resources/user-guides/How-to-mine-Monero-XMR-without-a-mining-equipment.md

```diff
@@ -1,49 +1,27 @@
-money to invest into building it, you can still mine Monero with NiceHash.
+Si no cuentas con un equipo de minado rentable, tiempo o dinero para invertir en construirlo, puedes minar Monero con NiceHash.
 
-NiceHash is a hashing power marketplace. Sellers of hashing
-power, i.e. miners, provide the hashing power for buyers (those who want to buy
-a service of mining a certain coin). Hence, NiceHash can provide you a massive
-hashing power in short amount of time. You won’t have to wait for years to see
-if you will make a profit or not and you can control which coin, at which pool,
-and for how long you want to mine.
+NiceHash es un marketplace de poder de minado. Vendedores de poder de hash, por ejemplo, mineros, proveen el poder de minado para compradores (aquellos que desean un servicio de minado de alguna moneda). Por lo tanto, NiceHash puede proveerte con un poder de hash masivo en un corto periodo de tiempo. No tendrás que esperar años para ver si lograrás ganancias o no y puedes controlar qué moneda, qué pool y por cuánto tiempo deseas minar.
 
-### **Step 1:** Create new account at NiceHash
+### **Paso 1:** Crear una nueva cuenta en NiceHash
 
-Visit [registration
-page](https://www.nicehash.com/?p=register) and register with your e-mail address.
+Visita la [página de registro](https://www.nicehash.com/?p=register) y únete con tu dirección de e-mail.
 
-### **Step 2:** Deposit some Bitcoins to your account
+### **Paso 2:** Deposita algo de Bitcoins a tu cuenta
 
-You will mine Monero, but you can buy hashing power at
-NiceHash only with Bitcoins. You can always withdraw unspent Bitcoins from your
-account back to any Bitcoin wallet.
+Minarás Monero, pero sólo puedes comprar poder de hash con Bitcoins. Siempre podrás retirar Bitcoins que no han sido gastados de tu cuenta a cualquier monedero de Bitcoin.
 
-Visit your [wallet
-page](https://www.nicehash.com/?p=wallet) and make a deposit. Note that the minimum price for placing an order
-equals 0.01 BTC.
+Ve a la [página de tu monedero](https://www.nicehash.com/?p=wallet) y haz un depósito. Ten en cuenta que el precio mínimo para hacer una orden es de 0.01 BTC.
 
-### **Step 3:** Find a suitable pool for mining and add it to your pool list
+### **Paso 3:** Busca una pool adecuada para minar y agrégala a tu lista
 
-Selection of the pool plays a big role in the final amount
-of mined cryptocurrency. Make sure the pool you have selected can handle
-massive hashing rate and loads of shares, especially from a single connection.
-You can find a list of Monero pools [here](https://bitcointalk.org/index.php?topic=583449.0).
+La selección de la pool juega un gran rol en la cantidad final de criptomoneda a minar. Asegúrate que pool que selecciones puede soportar poder de hash masivo y cargas de shares, especialmente desde una conexión individual. Puedes encontrar una lista de las pools de Monero [aquí](https://bitcointalk.org/index.php?topic=583449.0).
```

Comment:
"Asegúrate que **la/el** pool que selecciones **pueda** soportar el poder de hashes masivo (aqui siento que es mejor no traducirla para seguir la idea de que la cantidad de hashes que reciben los pools desde el poder de procesamiento de los equipos; si perdemos la coherencia, entonces, "...pueda soportar el poder de procesamiento y la carga de...") y la carga de shares (encontre esta definición para futuras traducciones si a alguien se le viene a la cabeza como traducirla [share](https://ethereum.stackexchange.com/questions/4529/for-pool-mining-what-exactly-is-a-share)), especialmente desde una conexión individual. Puedes...".